content/id/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace.md

```diff
@@ -0,0 +1,272 @@
+Bagaimana Kontainer mendapatkan nilai tersebut?
+
+```
+resources:
+  limits:
+    memory: 1Gi
+  requests:
+    memory: 1Gi
+```
+
+Karena Containermu tidak menentukan permintaan dan limit memori, Kontainer tersebut diberikan
+[permintaan dan limit memori semula](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/)
+dari LimitRange.
+
+Pada tahap ini, Kontainermu mungkin saja berjalan ataupun mungkin juga tidak berjalan. Ingat bahwa prasyarat
+untuk tugas ini adalah Node harus memiliki setidaknya 1 GiB memori. Jika tiap Node hanya memiliki
+1 GiB memori, maka tidak akan ada cukup memori untuk dialokasikan pada setiap Node untuk memenuhi permintaan 1 Gib memori. Jika ternyata kamu menggunakan Node dengan 2 GiB memori, maka kamu mungkin memiliki cukup ruang untuk memenuhi permintaan 1 GiB tersebut.
+
+Menghapus Pod:
+
+```
+kubectl delete pod constraints-mem-demo-4 --namespace=constraints-mem-example
+```
+
+## Pelaksanaan batasan minimum dan maksimum memori
+
+Batasan maksimum dan minimum memori yang yang ditetapkan pada sebuah Namespace oleh LimitRange dilaksanakan
+hanya ketika Pod dibuat atau diperbarui. Jika kamu mengubah LimitRange, hal tersebut tidak akan memengaruhi
+Pods yang telah dibuat sebelumnya.
+
+## Motivasi untuk batasan minimum dan maksimum memori
+
+Sebagai seorang administrator klaster, kamu mungkin ingin menetapkan pembatasan jumlah memori yang dapat digunakan oleh Pod.
+Sebagai contoh:
+
+* Tiap Node dalam sebuah klaster memiliki 2 GB memori. Kami tidak ingin menerima Pod yang meminta
+lebih dari 2 GB memori, karena tidak ada Node pada klater yang dapat memenuhi permintaan tersebut.
+
+* Sebuah klaster digunakan bersama pada departemen produksi dan pengembangan.
+Kamu ingin mengizinkan beban kerja (_workload_) produksi untuk menggunakan hingga 8 GB memori, tapi
+kamu ingin beban kerja pengembangan untuk terbatas cukup sampai 512 MB saja. Kamu membuat Namespace terpisah
+untuk produksi dan pengembangan, dan kamu menerapkan batasan memori pada tiap Namespace.
+
+## Bersih-bersih
+
+Menghapus Namespace:
+
+```shell
+kubectl delete namespace constraints-mem-example
+```
+
+
+
+## {{% heading "whatsnext" %}}
+
+
+### Untuk administrator klaster
+
+* [Configure Default Memory Requests and Limits for a Namespace](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/)
+
+* [Configure Default CPU Requests and Limits for a Namespace](/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace/)
+
+* [Configure Minimum and Maximum CPU Constraints for a Namespace](/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace/)
+
+* [Configure Memory and CPU Quotas for a Namespace](/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace/)
+
+* [Configure a Pod Quota for a Namespace](/docs/tasks/administer-cluster/manage-resources/quota-pod-namespace/)
+
+* [Configure Quotas for API Objects](/docs/tasks/administer-cluster/quota-api-object/)
```

Comment:
You can translate the page title also!

Reply:
Sure! I translated it in in e323b8c5920d6ae02c050306495e784b8e19ad9f